client/api_object.go

```diff
@@ -725,3 +728,69 @@ func (c *client) UpdateObjectVisibility(ctx context.Context, bucketName, objectN
 
 	return c.sendTxn(ctx, updateObjectMsg, opt.TxOpts)
 }
+
+// ListObjectsByObjectID list objects by object ids
+func (c *client) ListObjectsByObjectID(ctx context.Context, objectIds []uint64) (types.ListObjectsByObjectIDResponse, error) {
```

Comment:
what the difference between headObjectByID?

Reply:
One is an interface on the chain side, and the other is an interface provided by SP metadata for the Data Marketplace to use. Besides these, this interface can query multiple object information by passing in an object id list, while HeadObjectByID can only return information for a single object.

Reply:
please add more comment about the usage and workflow

Reply:
updated